mmhuman3d/data/data_converters/humman.py

```diff
@@ -0,0 +1,277 @@
+                    smpl_ = iphone_smpl
+
+                assert device_id >= 0, f'Negative device id: {device_id}'
+
+                keypoints_convention = smc_reader.get_keypoints_convention()
+                assert keypoints_convention_ == keypoints_convention
+
+                # get keypoints2d (all frames)
+                keypoints2d, keypoints2d_mask = smc_reader.get_keypoints2d(
+                    device, device_id)
+                if keypoints2d_mask_ is None:
+                    keypoints2d_mask_ = keypoints2d_mask
+                assert (keypoints2d_mask_ == keypoints2d_mask).all()
+                keypoints_2d_.append(keypoints2d)
+
+                # compute bbox from keypoints2d
+                xs, ys = keypoints2d[:, :, 0], keypoints2d[:, :, 1]
+                xmins, xmaxs = np.min(xs, axis=1), np.max(xs, axis=1)
+                ymins, ymaxs = np.min(ys, axis=1), np.max(ys, axis=1)
+
+                for xmin, xmax, ymin, ymax in zip(xmins, xmaxs, ymins, ymaxs):
+                    bbox_xyxy = [xmin, ymin, xmax, ymax]
+                    bbox_xywh = self._bbox_expand(bbox_xyxy, scale_factor=1.2)
+                    bbox_xywh_.append(bbox_xywh)
+
+                # get keypoints3d (all frames)
+                keypoints3d, keypoints3d_mask = smc_reader.get_keypoints3d(
+                    device, device_id)
+                if keypoints3d_mask_ is None:
+                    keypoints3d_mask_ = keypoints3d_mask
+                assert (keypoints3d_mask_ == keypoints3d_mask).all()
+
+                # root-align keypoints3d
+                left_hip_keypoints = keypoints3d[:, [self.left_hip_idx], :3]
+                right_hip_keypoints = keypoints3d[:, [self.right_hip_idx], :3]
+                root_keypoints = \
+                    (left_hip_keypoints + right_hip_keypoints) / 2.0
+                keypoints3d[..., :3] = keypoints3d[..., :3] - root_keypoints
+                keypoints_3d_.append(keypoints3d)
+
+                # get smpl (all frames)
+                smpl_dict = smc_reader.get_smpl()
+                smpl_['body_pose'].append(smpl_dict['body_pose'])
+                smpl_['global_orient'].append(smpl_dict['global_orient'])
+                smpl_['transl'].append(smpl_dict['transl'])
+
+                # expand betas
+                betas_expanded = np.tile(smpl_dict['betas'],
+                                         num_frames).reshape(-1, 10)
+                smpl_['betas'].append(betas_expanded)
+
+                # get image paths (smc paths)
+                image_path = os.path.basename(ann_path)
+                for frame_id in range(num_frames):
+                    image_id = (device, device_id, frame_id)
+                    image_id_.append(image_id)
+                    image_path_.append(image_path)
+
+                assert len(keypoints2d) == num_frames
+                assert len(keypoints3d) == num_frames
+
+        if not os.path.isdir(out_path):
+            os.makedirs(out_path)
```

Comment:
[`os.makedirs(out_path, exist_ok=True)`](https://docs.python.org/3.2/library/os.html#os.makedirs)

Reply:
Fixed in https://github.com/open-mmlab/mmhuman3d/pull/73/commits/f943649f4a1fd45aa0e9e7134989f0b4c127b31e.